### DISCUSSION TITLE
Create3 Cargo Bay Depth?

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#473] [Questions] [answered]
### How are you connecting to your Create 3?

I'm not sure

### Computer(s) Model(s) and Operating System(s)

n/a

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

n/a

### Which RMW is your robot running?

I don't know

### Does your robot have an assigned namespace? If so, please share.

n/a

### Is the robot connected to a network? If so, what is the network type?

n/a

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

The Create3 Mechanical page includes a dimensioned top view of the Create3 Cargo Bay, but no side view to know the depth in the mounting plate area.  The dimensions I am able to guess from the drawing:
- Mounting area: 158mm (6.22") wide by 113.6mm (4.47") by ??
- Max Inside Dim: 158mm (6.2") x 158mm (6.2") x ??

but I cannot find a depth in the mounting area?

Ref: [Cargo Bay - Top](https://iroboteducation.github.io/create3_docs/hw/mechanical/#cargo-bay-top)

**Comment by brianabouchard** [comment, marked as answer]
Hi @slowrunner! I pulled the STEP file provided on that page into some CAD software and pulled the dimensions you are interested in. Let me know if you want any additional dimensions. Note all dimensions provided are in mm and depending on the size of your component, the usable internal width is less than 158 mm due to the supports on the sides. I've listed that dimension for you as well. 
<img width="1141" alt="Cargo Bay Assembly" src="https://github.com/iRobotEducation/create3_docs/assets/66318600/cda08dff-52bf-457f-98c8-30ba0065972b">

**Reply by slowrunner** [reply]
Wow! Exactly my thoughts on how to get at this, but I didn't know how to view and measure the step file I found in the repo.  (You might think with a 1976 BS Mechanical Engineering degree, that it would have been my thing, but I changed to software a year after graduating.  I'm danger to myself with tools, and CAD brings on trauma! )

Thank you so much.

CARGO BAY
- Depth 38.95mm (1.53")
- Mounting area: 142.23mm (5.6") wide x 112.75mm (4.44")
- Total usable inside dimension: 142.23mm (5.6") x 160.7mm (6.3")

The purpose for wanting to know is to see if [the 12v powered USB hub with 121.2 x 35 x 18.8 mm（4.77 x 1.38 x 0.74 inch）dimension](https://github.com/iRobotEducation/create3_docs/discussions/127#discussioncomment-3323026) would fit inside the cargo bay turned on its side so the connectors face horizontal.  Trying to figure out an off the shelf power distribution for my Oak-D-Lite (and LIDAR if the 3D point cloud stuff does not provide enough precision in SLAM localization).  I want to isolate the accessories from the Pi5 5v supply to allow full processor load (7.2W 1.4A during full stress, 3W 0.6A idle).  My prior Raspberry Pi boards have sometimes struggled to hold the 5v above the low voltage limit under heavy load conditions.  I really want to be able to utilize the 3x ROS processing capability the Pi5 can deliver (compared with Pi4).

**Reply by brianabouchard** [reply]
Hahaha! As a MechE, I was pretty excited to be able to send over a drawing. 

Can't wait to see what you come up with for this power option!
